### ISSUE TITLE
Duplicated IDs

### ISSUE THREAD

**Issue by afercia** [obenland/shiny-updates#147]
Noticed in the `update-core.php` screen, not sure if it happens also on other screens. some elements have duplicated IDs for example different buttons or hidden fields and their related buttons.
Please consider I really don't know anything about the Shiny Updates code and the reason for this but duplicated IDs should be preferably avoided.
Up to you as you are the only ones who can evaluate this. Some examples:

```
<button type="submit" name="upgrade" id="upgrade" class="button update-link">Update</button>
<button type="submit" name="upgrade" id="upgrade" class="button update-link">Re-install Now</button>
```
or
```
<input type="hidden" name="checked[]" id="checkbox_fc664c683b3272a1dba04989531a47e8" value="twentysixteen">
<button type="submit" name="checkbox_fc664c683b3272a1dba04989531a47e8" id="checkbox_fc664c683b3272a1dba04989531a47e8" class="button update-link">Update</button>
```

**Comment by swissspidy** [comment]
I'd definitely consider this a bug.

The `upgrade` ID isn't really needed as all. The `checkbox…` stuff is a leftover from the current code in WordPress where there are checkboxes for bulk updates, but it looks like we don't need them either.

Will remove them after a round of testing.
